# MPC parameters
state_costs:
  x: 1.0e+6
  dx: 1.0e+1
  z: 5.0e+5
  dz: 1.0e+6
  att: 1.0e-3
  datt: 1.0e-0
  droll: 1.0e-2

input_costs:
  servo: 5.0e-1
  thrust: 1.0e-1
  torque: 1.0e-1

# constraints
min_z: -1
min_dz: -0.4

max_dx: 1.2
max_dz: 2
max_datt: 0.6
max_attitude_angle: 45

# scaling
scaling_x: 2
scaling_z: 4

horizon_length: 1.0

max_iter: 1
line_search_max_iter: 5
mpc_period: 0.02
feedforward_period: 0.02

